### DISCUSSION TITLE
Other coding applications that are useful

### DISCUSSION THREAD

**Discussion by dsk114u** [community/community#30340] [New to GitHub]
Is there any othe coding apps that you feel are equal to or better than GitHub?  They can b from anywhere and if so is there a path to reach them?

**Comment by Spacexowboy** [comment]
Unsubscribe

**Comment by NikhilMath** [comment]
Git Kraken 
VS Code
iTerm2
xCode

**Reply by NikhilMath** [reply]
that is the coding stuff I use.
